test/Scientist.Test/ScientistTests.cs

```diff
@@ -525,5 +525,72 @@ public void ExperimentIgnoredWithComplexIgnoreFunc()
             Assert.True(experimentResult.IgnoredObservations.Any());
             Assert.True(experimentResult.Matched);
         }
+
+        [Fact]
+        public void SingleContextIncludedWithPublish()
+        {
+            var mock = Substitute.For<IControlCandidate<int>>();
+            mock.Control().Returns(42);
+            mock.Candidate().Returns(42);
+
+            const string experimentName = nameof(SingleContextIncludedWithPublish);
+
+            var result = Scientist.Science<int>(experimentName, e =>
+            {
+                e.Use(mock.Control);
+                e.Try(mock.Candidate);
+                e.Context("test", "data");
+            });
+
+            var publishResults = TestHelper.Results<int>().First(m => m.ExperimentName == experimentName);
+
+            Assert.Equal(42, result);
+            Assert.True(publishResults.Contexts.Any());
```

Comment:
Should this verify the contents of `Contexts` instead of just making sure something exists?